AdyenCard/Components/Card/CardComponent.swift

```diff
@@ -128,6 +128,8 @@ public class CardComponent: PresentableComponent,
     public var requiresModalPresentation: Bool { storedCardComponent?.requiresModalPresentation ?? true }
     
     public func stopLoading() {
+        // don't access cardViewController when it's storedCardComponent
```

Comment:
Can we add more context? Like we can extract the guard check to a separate method returning bool and give it a meaningful name. Now this check seems to be pure magic.